pkg/producer/convertor/test/flowtype_test.go

```diff
@@ -26,74 +26,7 @@ import (
 	"github.com/vmware/go-ipfix/pkg/producer"
 	"github.com/vmware/go-ipfix/pkg/producer/convertor"
 	"github.com/vmware/go-ipfix/pkg/registry"
-)
-
-var (
-	// Hard coding the kafka msg in bytes. Need to figure out a way to generate this.
-	msg1ForFlowType1 = []byte{0x0, 0x0, 0x0, 0xa4, 0x10, 0x1, 0x18,
-		0xd2, 0x9, 0x32, 0x8, 0x31, 0x30, 0x2e, 0x30, 0x2e, 0x30, 0x2e, 0x31, 0x3a,
-		0x8, 0x31, 0x30, 0x2e, 0x30, 0x2e, 0x30, 0x2e, 0x32, 0x40, 0xd2, 0x9, 0x48,
-		0xae, 0x2c, 0x50, 0x6, 0x58, 0xe8, 0x7, 0x60, 0xe8, 0x7, 0x68, 0xe8, 0x7,
-		0x70, 0xe8, 0x7, 0x78, 0x32, 0x80, 0x1, 0x32, 0x88, 0x1, 0x32, 0x90, 0x1,
-		0x32, 0x9a, 0x1, 0x5, 0x4, 0x70, 0x6f, 0x64, 0x31, 0xa2, 0x1, 0x7, 0x6, 0x70,
-		0x6f, 0x64, 0x6e, 0x73, 0x31, 0xaa, 0x1, 0x6, 0x5, 0x6e, 0x6f, 0x64, 0x65,
-		0x31, 0xb2, 0x1, 0x5, 0x4, 0x70, 0x6f, 0x64, 0x32, 0xba, 0x1, 0x7, 0x6, 0x70,
-		0x6f, 0x64, 0x6e, 0x73, 0x32, 0xc2, 0x1, 0x6, 0x5, 0x6e, 0x6f, 0x64, 0x65,
-		0x32, 0xca, 0x1, 0xb, 0x31, 0x39, 0x32, 0x2e, 0x31, 0x36, 0x38, 0x2e, 0x30,
-		0x2e, 0x31, 0xd2, 0x1, 0x5, 0x4, 0x73, 0x76, 0x63, 0x31, 0xea, 0x1, 0x1,
-		0x0, 0xf2, 0x1, 0x1, 0x0, 0xfa, 0x1, 0x1, 0x0, 0x82, 0x2, 0x1, 0x0, 0x8a,
-		0x2, 0x9, 0x31, 0x32, 0x37, 0x2e, 0x30, 0x2e, 0x30, 0x2e, 0x31, 0x90, 0x2,
-		0x83, 0x25}
```

Comment:
were these moved for consumer tests?

Reply:
yes, planning to add some testing code for consumer.